[2025-06-13T23:03:17.847+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_projeto.extract_data manual__2025-06-13T23:03:14.027463+00:00 [queued]>
[2025-06-13T23:03:17.858+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_projeto.extract_data manual__2025-06-13T23:03:14.027463+00:00 [queued]>
[2025-06-13T23:03:17.859+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-06-13T23:03:17.873+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): extract_data> on 2025-06-13 23:03:14.027463+00:00
[2025-06-13T23:03:17.877+0000] {standard_task_runner.py:60} INFO - Started process 303 to run task
[2025-06-13T23:03:17.880+0000] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'etl_projeto', 'extract_data', 'manual__2025-06-13T23:03:14.027463+00:00', '--job-id', '54', '--raw', '--subdir', 'DAGS_FOLDER/etl_pipeline.py', '--cfg-path', '/tmp/tmporrjjk2a']
[2025-06-13T23:03:17.883+0000] {standard_task_runner.py:88} INFO - Job 54: Subtask extract_data
[2025-06-13T23:03:17.947+0000] {task_command.py:423} INFO - Running <TaskInstance: etl_projeto.extract_data manual__2025-06-13T23:03:14.027463+00:00 [running]> on host a984848cc8d0
[2025-06-13T23:03:18.054+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='kayo' AIRFLOW_CTX_DAG_ID='etl_projeto' AIRFLOW_CTX_TASK_ID='extract_data' AIRFLOW_CTX_EXECUTION_DATE='2025-06-13T23:03:14.027463+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-06-13T23:03:14.027463+00:00'
[2025-06-13T23:03:20.863+0000] {logging_mixin.py:188} INFO - Arquivo salvo em /opt/airflow/data/dados_api.csv
[2025-06-13T23:03:20.866+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-06-13T23:03:20.890+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=etl_projeto, task_id=extract_data, execution_date=20250613T230314, start_date=20250613T230317, end_date=20250613T230320
[2025-06-13T23:03:20.959+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-06-13T23:03:20.993+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
